internal/faults/common.go

```diff
@@ -0,0 +1,278 @@
+// Package faults provides common headers and client-side fault injection
```

Comment:
A better name than `common.go` could be `faults.go`. It's a nice convention to describe the main entry point for a package and `common` or `util` tend to get overused as catch-all file.

Reply:
Done.